ironic/resource_ironic_node_v1.go

```diff
@@ -628,3 +666,40 @@ func (workflow *provisionStateWorkflow) changeProvisionState(target nodes.Target
 
 	return false, nodes.ChangeProvisionState(workflow.client, workflow.d.Id(), *opts).ExtractErr()
 }
+
+// Call Ironic's API and change the power state of the node
+func changePowerState(client *gophercloud.ServiceClient, d *schema.ResourceData, target nodes.TargetPowerState) error {
+	timeout, err := strconv.Atoi(d.Get("power_state_timeout").(string))
+	if err != nil {
+		timeout = 300
```

Comment:
Looks like you're silently ignoring errors here, is it a good idea?

Reply:
Fixed

Reply:
I've updated this code to leave this unspecified if the user didn't give us a timeout.  

I still needed some value to wait for how long to poll Ironic for target_provision_state to clear, so there's still a default value here.  There's no other way to know if the request is finished, right?